src/docs/product/integrations/integration-platform/ui-components/index.mdx

```diff
@@ -5,348 +5,40 @@ redirect_from:
-- `installationId` - Your integration's installation ID (helps you determine the requesting Sentry org)
-- `projectSlug` - slug of the project the issue belongs to
-- `filename` - full path of the stack frame file
-- `lineNo` - line number of the stack trace in the file
-
-### Example
-
-```javascript
-"elements": [
-    {
-        "type": "stacktrace-link",
-        "uri": "/stacktrace-redirect",
-    }
-]
-```
-
-Any time a component supports a `uri` attribute, Sentry will make a request to the third-party service using the Base URL or their `webhook_url`.
-
-For a more complete description of the grammar, see the [source code](https://github.com/getsentry/sentry/blob/master/src/sentry/api/validators/sentry_apps/schema.py).
-
-## Alert Rule Action
-
-The Alert Rule Action component allows the developer to get parameters to define routing or configuration in alert rules for their service. When alert rules are triggered for a user, the configured service will receive [Issue Alert](/product/integrations/integration-platform/webhooks/#issue-alerts) and [Metric Alert](/product/integrations/integration-platform/webhooks/#metric-alerts) webhook events with the specified settings for that service to further route the alert.
-
-### Schema
-
-```javascript
- {
-    "type": "alert-rule-action",
-    "title": <String>,
-    "settings": {
-        "type": "alert-rule-settings",
-        "uri": <URI>,
-        "required_fields": <Array<FormField>>,
-        "optional_fields": <Array<FormField>>,
-        "description": <String>
-    }
-}
-```
-
-### Attributes
-
-- `title` - (Required) The title shown in the UI component.
-- `uri` - (Required) Sentry will make a POST request to the URI when the User submits the form. If the services fails to process the request (status code >= 400), this component will bubble up the error to the User with the provided response text.
-- `required_fields` - (Required) List of `FormField` components the User is required to complete.
-- `optional_fields` - List of optional `Form Field` components the User may complete.
-- `description` - Optional text that will be displayed above the form. Limited to 140 characters.
-
-### Example
-
-```javascript
-
-"elements": [
+  "elements": [
     {
-        "type": "alert-rule-action",
-        "title": "Send Alerts to",
-        "settings": {
+      "type": "alert-rule-action",
+      "title": "Notify my a channel in my app",
+      "settings": {
         "type": "alert-rule-settings",
         "uri": "/sentry/alert-rule",
-            "required_fields": [
-                {
-                    "type": "text",
-                    "label": "Channel",
-                    "name": "channel"
-                }
-            ]
-        }
+        "required_fields": [
+          {
+            "type": "text",
+            "label": "Which channel should receive the alert?",
+            "name": "channel"
+          }
+        ]
+      }
     }
```

Comment:
Instead of 
```
// Each element is a specification for a component the developer wants to use.
...
```
we could say 
```
// Component schemas to be used by the integration
// e.g. { "type": "alert-rule-action", ... }
...
```